squad/ci/templates/squad/ci/admin_diff.html

```diff
@@ -0,0 +1,49 @@
+{% load squad %}
+<html>
+  <head>
+    <style type='text/css'>
+body {
+  font-family: sans-serif;
+  font-size: 12px;
+}
+h1 {
+  border-bottom: 1px solid #babdb6;
+}
+table.table-bordered {
+  border-collapse: collapse;
+}
+table.table-bordered td,
+table.table-bordered th {
+  border: 1px solid #2e3436;
+  padding: 0.25em;
+}
+table.table-bordered td.pass {
+  background: #8ae234;
+  color: #214203;
+}
+table.table-bordered td.fail {
+  background: #ef2929;
+  color: #260000;
+}
+.footer {
+  padding: 0.25em;
+  margin-top: 2em;
+}
+.footer, .footer a:link, .footer a:visited {
+  color: #babdb6;
+}
+    </style>
+  </head>
+  <body>
+    <h1>Failed to retrieve test job</h1>
```

Comment:
this is not actually what happened. it's not that we failed to retrieve the test job, but that the test job itself failed.